docs/test-cases.md

```diff
@@ -12,19 +12,24 @@ Test cases objects are interpreted by the framework to build up the mock and run
 
 ## TestCase
 
-TestCase defines each test that will be run sequentially.
+[TestCase](https://github.com/getyourguide/istio-config-validator/blob/195017ef364f89b773492f2108e4478188a754ff/internal/pkg/parser/testcase.go#L32-L42) defines each test that will be run sequentially.
 
 | Field       | Type                                                                                                            | Description                                                |
 |-------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|
 | description | string                                                                                                          | Short description of what the testing is about.            |
 | wantMatch   | bool                                                                                                            | If the test case should assert `true` or `false`           |
 | request     | [request](#Request)                                                                                                         | Crafted requests that will mocked against VirtualServices  |
 | route       | [HTTPRouteDestination[]](https://istio.io/docs/reference/config/networking/virtual-service/#HTTPRouteDestination) | Route destinations that will be asserted for each request. |
+| redirect    | [HTTPRedirect](https://istio.io/latest/docs/reference/config/networking/virtual-service/#HTTPRedirect)          | Any redirect logic to test
```

Comment:
This wasn't implemented. Fixed in https://github.com/getyourguide/istio-config-validator/pull/420

Reply:
Should I remove it from the docs for now or wait until #420 is merged?